src/Entities/NotionParent.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace FiveamCode\LaravelNotionApi\Entities;
+
+use FiveamCode\LaravelNotionApi\Exceptions\HandlingException;
+
+/**
+ * Class NotionParent.
+ */
+class NotionParent extends Entity
```

Comment:
Maybe rename this to "Parent"?  We don't have a Parent class yet, so would be possible.